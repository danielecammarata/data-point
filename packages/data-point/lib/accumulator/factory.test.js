/* eslint-env jest */

const Factory = require("./factory");

describe("Accumulator", () => {
  it("should create base accumulator object", () => {
    expect(new Factory.Accumulator()).toMatchSnapshot();
  });
});
