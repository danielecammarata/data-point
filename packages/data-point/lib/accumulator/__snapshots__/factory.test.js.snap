// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accumulator should create base accumulator object 1`] = `
Accumulator {
  "context": undefined,
  "locals": undefined,
  "reducer": undefined,
  "trace": false,
  "traceGraph": Array [],
  "value": undefined,
  "values": undefined,
}
`;
